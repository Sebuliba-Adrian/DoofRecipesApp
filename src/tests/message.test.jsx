import React from "react";
import renderer from "react-test-renderer";
import Message from "../components/message";

const props = {
  message: ""
}
it("renders correctly", () => {
  const message= renderer.create(<Message {...props} />).toJSON();
  expect(message).toMatchSnapshot();
});
