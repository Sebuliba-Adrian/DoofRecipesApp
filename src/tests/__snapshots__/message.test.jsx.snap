// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="message alert alert-dismissible fade show alert-info no-border-corners"
  role="alert"
>
  <button
    className="close fa fa-close align-middle"
    data-dismiss="alert"
    type="button"
  />
  
</div>
`;
